sdk/appcenter-crashes/src/androidTest/java/com/microsoft/appcenter/crashes/CrashesAndroidTest.java

```diff
@@ -571,4 +564,87 @@ public void run() {
         assertNotNull(files);
         assertEquals(2, files.length);
     }
+
+    @Test
+    public void getMinidumpSubfolder() {
+        File newMinidumpSubfolder = ErrorLogHelper.getNewMinidumpSubfolder();
+        assertTrue(newMinidumpSubfolder.exists());
```

Comment:
Do we need to check some names here?

Reply:
I don't think so, we have no folder-name bindings.